### PR TITLE
Feature/show occupied

### PR DESCRIPTION
**Pull request by Earlo**

### CODE REVIEW

**Comment by VonLatvala** [comment]
This commit also brings in prettier

File: src/containers/UserForm.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { connect } from "react-redux";
+import { submitUserInfoAction } from "../actions/submitUserInfoAction";
+import { updateUserInfoFormAction } from "../actions/updateUserInfoFormAction";
+import UserForm from "../components/UserForm";
+
+const mapStateToProps = state => ({
+    info: state.userReducer.info || {},
+    form: state.userReducer.form || {},
```
Comment (VonLatvala) [inline src/containers/UserForm.js]: is setting these defaults here a thing? Shouldn't the store/reducers handle this instead?

**Review by VonLatvala — APPROVED** [review]
Check the comment

File: src/containers/UserForm.js (hunk 1 above)
Comment (Earlo) [reply src/containers/UserForm.js]: Eeh, yeah, hmm, I guess it could be avoided

**Pull request merged**